tests/playground.html

```diff
@@ -67,6 +67,7 @@
 <script src="blocks/test_blocks.js"></script>
 <script src="themes/test_themes.js"></script>
 <script src="./playgrounds/screenshot.js"></script>
+<script src="./test_blocks_toolbox.js"></script>
```

Comment:
Since this toolbox is only for the playground, let's move to ``tests/playgrounds``.